### DISCUSSION TITLE
Spacing between lines

### DISCUSSION THREAD

**Discussion by augustocdias** [wezterm/wezterm#2668] [Q&A]
How do I decrease the spaces between lines without making them overlap each other?

I'm quite satisfied with the way it looks by default, but if you check the vertical lines used to highlight the indentation, you can see there's some space between them, whereas with iterm it looks the way I'd like. Both are FiraCode 12 but for italics I set a different font in Wez.
![image](https://user-images.githubusercontent.com/1080789/197970483-9c529691-5341-41db-b4f3-903ad4e47e01.png)

I've tried to play with the `line_height` setting but as you can see the fonts start to overlap each other.

![image](https://user-images.githubusercontent.com/1080789/197971082-661440b8-9104-4e02-8e24-0eaf8366a8ca.png)

**Comment by wez** [comment]
Take a look at: https://wezfurlong.org/wezterm/config/lua/wezterm/font_with_fallback.html#dealing-with-different-fallback-font-heights

**Reply by augustocdias** [reply]
Unfortunately I have seen no difference and playing with the manual I could achieve only the same result as changing only the line_height
